rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnNeverTypeRector/Fixture/die_some.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnNeverTypeRector\Fixture;
+
+final class DieSome
+{
+    public function run()
+    {
+        echo 100;
+        die;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnNeverTypeRector\Fixture;
+
+final class DieSome
+{
+    public function run(): never
```

Comment:
when this rector would use `@return never` instead of the native return type, it would ease usage with php 7.x (without the need for a additional downgrading step) - would you accept a PR changing this rector from `:never` to `@return never` ?

Reply:
I think other rules also use `@x` alternative, unless the type is nativelly supported in PHP, so yes, go for it :+1:

Reply:
here we go https://github.com/rectorphp/rector-src/pull/296